apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.css

```diff
@@ -18,7 +18,7 @@
 }
 
 .pl-multiple-choice-dropdown .ts-wrapper .ts-control .item {
-  min-width: var(--pl-multiple-choice-dropdown-width, 220px);
+  min-width: var(--pl-multiple-choice-dropdown-width, 50px);
```

Comment:
@eliotwrobson are you at all concerned about backwards compatibility here? IMO this causes problems when there's a mix of short and long answers. Consider the demo question; if we remove the `size` attribute and then select a short answer, longer answers will be displayed with aggressive wrapping:

<img width="134" alt="Screenshot 2024-09-30 at 13 54 59" src="https://github.com/user-attachments/assets/1328b604-b8e5-474b-9436-89a716e1776d">

This would be even worse if the answer was a longer word or phrase.

I'm wondering if we should leave the default alone and rely on folks setting `size` if they really want to change the width.

Reply:
Is the consensus then that I should remove the change to CSS? 

If so, should an additional note be made somewhere in the documentation stating that size can be used to shrink to fit the content if required? Or is the existing documentation sufficient?

Reply:
Yes, that's the consensus. The existing documentation ("Manually set the size of the dropdown to a fixed width") is completely fine; there is zero mention of it being used only to _increase_ the width.

Reply:
Looks like this was resolved in 4b68a08.